lua/bufferline/buffer.lua

```diff
@@ -50,11 +39,11 @@ local function get_name(opts, number)
       name = '[buffer ' .. number .. ']'
     end
   else
-    local buftype = nvim_buf_get_option(number, 'buftype')
+    local buftype = vim.bo[number].buftype
```

Comment:
`vim.bo[<number>].<option>` is a more concise way to get buffer options IMO but I'm not married to it, I can change them back.

Reply:
I prefer using the neovim API rather than using `vim.bo`, if you don't have a preference.